lectures/delta-time/README.md

```diff
@@ -0,0 +1,366 @@
+<h2 align=center>Lecture 03</h2>
+
+<h1 align=center>Delta Time</h1>
+
+<h3 align=center>9 Pegasus Moon, Imperial Year MMXXIV</h3>
+
+<p align=center><strong><em>Song of the day</strong>: <a href="https://youtu.be/f_G43LhHEos"><strong><u>Burgundy Red</u></strong></a> by 落日飛車 (SUNSET ROLLERCOASTER) (2016)</em></p>
```

Comment:
Change to:

<p align=center><strong><em>Song of the day</strong>: <a href="https://youtu.be/u9Di0D1DQps?si=Gmuom5tqc9ckMbZO"><strong><u>Pump! (feat. chelmico) [live]</u></strong></a> by PASOCOM MUSIC CLUB (2023)</em></p>

whenever you can!